ChatSecure/Classes/OTRAppDelegate.m

```diff
@@ -569,7 +569,9 @@ - (void) userNotificationCenter:(UNUserNotificationCenter *)center willPresentNo
 
 - (void) userNotificationCenter:(UNUserNotificationCenter *)center didReceiveNotificationResponse:(UNNotificationResponse *)response withCompletionHandler:(void (^)())completionHandler {
     NSDictionary *userInfo = response.notification.request.content.userInfo;
-    if (userInfo[kOTRNotificationThreadKey] == nil) {
+    if ([kOTRNotificationTypeNone isEqualToString:userInfo[kOTRNotificationType]]) {
+        // Nothing
+    } else if ([kOTRNotificationTypeSubscriptionRequest isEqualToString:userInfo[kOTRNotificationType]]) {
```

Comment:
Ok, fair enough, change submitted! Just my background in other languages showing through - "sending a message to nil" is to me a pretty outrageous operation... =)